_posts/ref/2022-01-18-wire-v1.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: page
+title:  "WIRE-V1 REST API and MQTT Request Handler"
+categories: [ref]
+---
+
+The WIRE-V1 cloud request handler and REST APIs allow to perform update, delete and get operations on the current Wire Graph. This request handler provides the equivalent of the `org.eclipse.kura.wire.graph.WireGraphService` Java API.
+
+WIRE-V1 is compatible with the Wire Graph import/export functionality of ESF Web UI, the get and update operations respectively consume and produce a JSON object specified by [ComponentConfigurationList]({{ site.baseurl }}{% link _posts/ref/2021-12-13-conf-v2.md %}#componentconfigurationlist).
```

Comment:
> WIRE-V1 is compatible with the Wire Graph import/export functionality of ESF Web UI, the get and update operations respectively consume and produce a JSON object specified by [ComponentConfigurationList]({{ site.baseurl }}{% link _posts/ref/2021-12-13-conf-v2.md %}#componentconfigurationlist).

The WIRE-V1 request handler is compatible with the Wire Graph import/export functionality of the ESF Web UI. In fact, the get and update operations respectively consume and produce a JSON object specified by [ComponentConfigurationList]({{ site.baseurl }}{% link _posts/ref/2021-12-13-conf-v2.md %}#componentconfigurationlist).